### ISSUE TITLE
Any scope for a responsive border styles module?

### ISSUE THREAD

**Issue by wdmtech** [basscss/borders#2]
Needed these in a project, so I made a package :smile: 

https://github.com/wdmtech/basscss-responsive-borders
https://www.npmjs.com/package/basscss-responsive-borders

If there's any interest, I'm up for filling out the rest of the breakpoint classes too. PRs welcome, of course.

**Comment by jxnblk** [comment]
I don't usually find any need for this sort of thing, so that's probably why it doesn't exist :)

That said, the naming convention you're using doesn't follow Basscss. Any reason for that?

**Comment by wdmtech** [comment]
Fair enough - How is it not follow naming conventions? I'm willing to change them If I'm going against the grain here :)

**Comment by jxnblk** [comment]
See the naming conventions here: https://github.com/basscss/borders/blob/master/index.css
To be consistent, it would be names like `md-border-top`.

**Comment by wdmtech** [comment]
Cool, makes sense - Not sure why I ended up using sm-bl instead of sm-border-left...! I'll get those changed.

**Comment by wdmtech** [comment]
Have updated the classnames accordingly.
